### PR TITLE
Remove bad tag from layout example

### LINKED ISSUE #2035: Invalid URL from MD docs
At the [Markdown page](https://docs.astro.build/en/guides/markdown-content/#frontmatter-layout) of the documentation there's a snippet of code to try out the .md files with Astro.

I've set up the files like they are in the documentation
- src/pages/mark.md
- src/layouts/MarkLayout.astro

and I've copy-pasted the snippets given in both. I'm leaving the code for each below.

mark.md:
```
---
layout: ../layouts/MarkLayout.astro
title: Hellp, World!
author: John Doe
date: "09 Aug 2022"
---

# Hi there!

This is a test page.

This is your first markdown page. It probably isn't styled much, although
Markdown does support **bold** and _italics._

To learn more about adding a layout to your page, read the next section on **Markdown Layouts.**
```

MarkLayout.astro (without comments):
```
---
import type { MarkdownLayoutProps } from 'astro';

type Props = MarkdownLayoutProps<{
	title: string;
	author: string;
	date: string;
}>;
const { frontmatter, url } = Astro.props;
---


<html>
  <head>
    <meta rel="canonical" href={new URL(url, Astro.site).pathname} />
    <title>{frontmatter.title}</title>
  </head>
  <body>
    <h1>{frontmatter.title} by {frontmatter.author}</h1>
    <slot />
    <p>Written on: {frontmatter.date}</p>
  </body>
</html>
```

but there seems to be a problem with the `href` attribute because after using `npm run dev` and looking up `localhost:3000:mark` gives out this error:
image

If I were to omit the `href` attribute of the `meta` tag, the page loads fine.
image

Here I've also added
```
<br />md url: {url} |<br />astro site: {Astro.site} |
```
after the date so I could see what the website was getting for each of the components of `new URL` used in the docs.

PS: My editor (and a quick search) points out that there shouldn't be a `rel` and an `href` attributes inside a `meta` tag so I would consider swapping the `meta` for a `link` tag, which is fine inside of `head` and obviously accepts `rel` and `href`.

### PR DESCRIPTION
**Pull request by Astisme**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)
- Translated content
- Changes to the docs site code

#### Description

- Closes #2035 
- What does this PR change? I've changed the `meta` tag for en, es, pt-br, ru to `link` (the other pages didn't have it). I've also added a block of code for the en page that specifies to the reader that he needs to check the `astro.config.mjs` file.
- Did you change something visual? No

### CODE REVIEW

File: src/pages/en/guides/markdown-content.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ const { frontmatter, url } = Astro.props;
 ---
 <html>
   <head>
-    <meta rel="canonical" href={new URL(url, Astro.site).pathname}>
+    <link rel="canonical" href={new URL(url, Astro.site).pathname}>
```
Comment (delucis) [inline src/pages/en/guides/markdown-content.md:99]: I actually think we’re safer to remove this entirely. It’s not an essential part of this example, and we don’t want to distract from the main focus of showing how layouts work by needing to explain this detail.

```suggestion
```

File: src/pages/en/guides/markdown-content.md [hunk 2]
```diff
@@ -106,6 +106,15 @@ const { frontmatter, url } = Astro.props;
...
 ```
+Be sure to have the `site` variable in your `astro.config.mjs` file.
+```
+...
+export default defineConfig({
+	...
+	site: "http://localhost:3000",
+	...
+});
+```
```
Comment (delucis) [inline src/pages/en/guides/markdown-content.md:117]: In which case we can remove this too. Sorry! 😅 

````suggestion
Be sure to have the `site` variable in your `astro.config.mjs` file.
```
...
export default defineConfig({
	...
	site: "http://localhost:3000",
	...
});
```
````

**Review by delucis — COMMENTED** [review]
Thank you for taking the time to contribute @Astisme! I think we can simplify this, so have made a suggestion here. Let me know what you think.

**Comment by Astisme** [comment]
Oh ok! No problem then ;)

**Review by delucis — APPROVED** [review]
Thanks @Astisme!

I’ve removed the changes to the `/pt-br/` and `/ru/` pages because we have a system for tracking translations and these pages are currently out of date: https://github.com/withastro/docs/issues/438. Changing them in this PR would mark them as updated which isn’t really accurate, so we usually leave changes in languages other than English to our translators rather than updating everything together in one PR. (Spanish in this example we can do because the translation is up-to-date.)

Welcome to Astro’s Docs Team 🥳 🚀

**Pull request merged**